### PR TITLE
fix incorrect exclusion of json_run_localhost in TSAN

### PR DESCRIPTION
**Pull request by billfeng327**
TSAN and ASAN were incorrectly disabled for json_run_localhost in PR [16568](https://github.com/grpc/grpc/pull/16568), they are now re-enabled

### CODE REVIEW

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
Objective-C binary sizes
*****************STATIC******************
  New size                      Old size
 2,019,040      Total (=)      2,019,040

 No significant differences in binary sizes

***************FRAMEWORKS****************
  New size                      Old size
11,179,732      Total (<)     11,179,738

 No significant differences in binary sizes


```

**Comment by grpc-testing** [comment]
```
Corrupt JSON data (indicates timeout or crash): 
    bm_call_create.BM_IsolatedFilter_ClientChannelFilter_NoOp_.counters.new: 10
    bm_call_create.BM_IsolatedFilter_ClientChannelFilter_NoOp_.counters.old: 10


[microbenchmarks] No significant performance differences
```

**Comment by billfeng327** [comment]
Known failures:
[#17138](https://github.com/grpc/grpc/issues/17138)
[#16579](https://github.com/grpc/grpc/issues/16579)
[#17170](https://github.com/grpc/grpc/issues/17170)

@billfeng327 all those failures are pre-existing

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
Objective-C binary sizes
*****************STATIC******************
  New size                      Old size
 2,019,040      Total (=)      2,019,040

 No significant differences in binary sizes

***************FRAMEWORKS****************
  New size                      Old size
11,179,733      Total (<)     11,179,735

 No significant differences in binary sizes


```

**Comment by grpc-testing** [comment]
```
Corrupt JSON data (indicates timeout or crash): 
    bm_call_create.BM_IsolatedFilter_ClientChannelFilter_NoOp_.counters.new: 10
    bm_call_create.BM_IsolatedFilter_ClientChannelFilter_NoOp_.counters.old: 10


[microbenchmarks] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

File: tools/remote_build/rbe_common.bazelrc [hunk 1]
```diff
@@ -70,7 +70,7 @@ build:msan --crosstool_top=@com_github_bazelbuild_bazeltoolchains//configs/ubunt
 build:tsan --copt=-gmlt
 # TODO(jtattermusch): use more reasonable test timeout
 build:tsan --test_timeout=3600
-build:tsan --test_tag_filters=-qps_json_driver,-json_run_localhost
+build:tsan --test_tag_filters=-json_run_localhost
```
Comment (jtattermusch) [inline tools/remote_build/rbe_common.bazelrc]: this is re-enabling qps_json_driver, json_run_localhost not re-enable. Probably not intended.

**Comment by vjpai** [comment]
Removing myself from the reviewers list since I have nothing to add beyond what Jan said.

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
Corrupt JSON data (indicates timeout or crash): 
    bm_call_create.BM_IsolatedFilter_ClientChannelFilter_NoOp_.counters.new: 10
    bm_call_create.BM_IsolatedFilter_ClientChannelFilter_NoOp_.counters.old: 10


[microbenchmarks] No significant performance differences
```

**Pull request merged**